README.md

```diff
@@ -1 +1,187 @@
 # Work In Progress - Onfido Java Library
+
+The official Java library for integrating with the Onfido API.
+
+Documentation can be found at <https://documentation.onfido.com>
+
+This library is only for use on the backend, as it uses Onfido API tokens which must be kept secret. If you do need to collect applicant data in the frontend of your application, we recommend that you use one of [the Onfido SDKs](https://developers.onfido.com/sdks/).
```

Comment:
I think we should be OK without this disclaimer for Java: it was just Node we were concerned about, because we were worried about people using it on the frontend.